README.md

```diff
@@ -105,12 +118,19 @@ Tools:
 - [Konga CLI Tool](https://github.com/Floby/konga-cli)
 - [Kong on Tutum](https://github.com/Sillelien/docker-kong)
 - [Kong GUI in JS](https://github.com/rsdevigo/jungle)
-- [Kong GUI in Py](https://github.com/vikingco/django-kong-admin)
+- [Kong Admin in Python](https://github.com/vikingco/django-kong-admin)
```

Comment:
Kong Admin but others say Kong UI or Kong GUI. They should probably all say "Kong Admin UI"